library/model/src/main/java/org/cru/godtools/model/TranslationFile.kt

```diff
@@ -2,9 +2,9 @@ package org.cru.godtools.model
 
 import org.cru.godtools.model.Base.INVALID_ID
 
-class TranslationFile constructor(
+data class TranslationFile(
     val translationId: Long = INVALID_ID,
-    val filename: String? = null
+    val filename: String
```

Comment:
Can filename no longer default to ```null``` because we always default to English?

Reply:
the `TranslationFile` object tracks the fact that a specific translation depends on a file on disk. `filename` being null makes no sense in that object definition